src/main/java/pl/javastart/task/ProductServices.java

```diff
@@ -0,0 +1,29 @@
+package pl.javastart.task;
+
+public class ProductServices {
+    private static double basicVatRate = 23.0;
+    private static double loweredVatRate = 8.0;
+    private static double zeroVatRate = 0.0;

```

Comment:
Do takich rzeczy zazwyczaj używamy stałych. Co prawda jest to dalej w bootcampie, ale skoro używasz już static, to powinieneś zapisywać np. `private static final double BASIC_VAT_RATE = 23.0`